src/main/java/build/buildfarm/common/redis/RedisQueueFactory.java

```diff
@@ -23,9 +25,9 @@ public QueueInterface getQueue(String queueType, String name) {
     if (queueType == null) {
       return null;
     }
-    if (queueType.equalsIgnoreCase("regular")) {
+    if (queueType.equalsIgnoreCase(Queue.QUEUE_TYPE.standard.name())) {
```

Comment:
Can we also make `BalancedRedisQueue.queueType` also using the new `QUEUE_TYPE` type instead of `String`?